pkg/kubelet/kubelet.go

```diff
@@ -893,8 +894,17 @@ func (kl *Kubelet) getServiceEnvVarMap(ns string) (map[string]string, error) {
 	return m, nil
 }
 
-// Make the service environment variables for a pod in the given namespace.
-func (kl *Kubelet) makeEnvironmentVariables(ns string, container *api.Container) ([]string, error) {
+func (kl *Kubelet) getDownwardAPIEnvVarMap(pod *api.Pod) map[string]string {
+	return map[string]string{
+		"K8S_API_POD_NAME": pod.Name,
```

Comment:
@bgrant0607 would you go so far to say, certified fresh?
On Fri, Mar 27, 2015 at 11:43 PM Brian Grant <notifications@github.com>
wrote:

> In pkg/kubelet/kubelet.go
> <https://github.com/GoogleCloudPlatform/kubernetes/pull/6000#discussion_r27342066>
> :
>
> > @@ -893,8 +894,17 @@ func (kl *Kubelet) getServiceEnvVarMap(ns string) (map[string]string, error) {
> >  	return m, nil
> >  }
> >
> > -// Make the service environment variables for a pod in the given namespace.
> > -func (kl *Kubelet) makeEnvironmentVariables(ns string, container *api.Container) ([]string, error) {
> > +func (kl *Kubelet) getDownwardAPIEnvVarMap(pod *api.Pod) map[string]string {
> > +	return map[string]string{
> > +		"K8S_API_POD_NAME": pod.Name,
>
> Ok, that's a legit use case.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/GoogleCloudPlatform/kubernetes/pull/6000/files#r27342066>
> .
>